perfkitbenchmarker/pkb.py

```diff
@@ -179,13 +180,13 @@ def ValidateBenchmarkInfo(benchmark_info):
 
 def ListUnknownBenchmarks():
   """Identify invalid benchmark names specified in the command line flags."""
-  valid_benchmark_names = frozenset(benchmark.GetInfo()['name']
-                                    for benchmark in benchmarks.BENCHMARKS)
+  valid_windows_benchmark_names = frozenset(windows_benchmarks.VALID_BENCHMARKS)
+  valid_benchmark_names = frozenset(benchmarks.VALID_BENCHMARKS)
   valid_benchmark_sets = frozenset(benchmark_sets.BENCHMARK_SETS)
   specified_benchmark_names = frozenset(FLAGS.benchmarks)
 
-  return sorted((specified_benchmark_names - valid_benchmark_names) -
-                valid_benchmark_sets)
+  return sorted(((specified_benchmark_names - valid_benchmark_names) -
```

Comment:
This logic is getting increasingly convoluted. Is it really worth the effort to check this separately? Looks like benchmark_sets.GetBenchmarkFromFlags already throws "Invalid benchmark" errors for unrecognized ones, and the benefit of being able to report multiple invalid ones in one go seems marginal.